idd/Energy+.idd.in

```diff
@@ -99859,6 +99883,7 @@ Output:EnvironmentalImpactFactors,
        \key Timestep
        \key Hourly
        \key Daily
+       \key Annual
```

Comment:
If we add Annual here, should we also add Environment? Annual/Environment/RunPeriod are all synonymous.